File: client/src/app/_models/blogPost.ts
import { Member } from './member';

export interface BlogPost {
    id: number;
    title: string;
    content: string;
    photo: File;
    photoUrl: string;
    photoPublicId : string;
    appUserId?: number;
    appUser: Member;
    createdDate: Date;
}

File: API/Entities/BlogPost.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class BlogPost
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string PhotoUrl { get; set; }
        public string PhotoPublicId { get; set; }
        public AppUser appUser  { get; set; }
        public int AppUserId { get; set; }
        public DateTime CreatedDate { get; set; } 
        public bool Deleted { get; set; }
    }
}